notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -122,7 +122,7 @@
       "source": [
```

Comment:
<p>combine installs to single cell</p><p>Add -q option</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/780/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>done...</p>